src/App/VPSScan/Scan/ScotlandYard.hs

```diff
@@ -41,24 +45,28 @@ instance FromJSON ScanResponse where
   parseJSON = withObject "ScanResponse" $ \obj ->
     ScanResponse <$> obj .: "scanId"
 
-createScotlandYardScan :: (MonadIO m, Has Diagnostics sig m) => VPSOpts -> m ScanResponse
-createScotlandYardScan VPSOpts {..} = runHTTP $ do
-  let body = object ["organizationId" .= organizationID, "revisionId" .= revisionID, "projectId" .= projectID]; FossaOpts {..} = fossaInstance
-  let auth = authHeader fossaApiKey
+createScotlandYardScan :: (MonadIO m, Has Diagnostics sig m) => ScotlandYardOpts -> m ScanResponse
+createScotlandYardScan ScotlandYardOpts {..} = runHTTP $ do
+  let VPSOpts{..} = syVpsOpts
+  let FossaOpts{..} = fossa
+  let parsedOrgId = read $ unpack organizationId :: Int
```

Comment:
can we decode this to an `Int` upstream somewhere? `read` is dangerous, and passing around an invalid organization ID is awkward. we should parse it as soon as possible